LayoutKit/Views/ReloadableViewLayoutAdapter+UICollectionView.swift

```diff
@@ -12,15 +12,15 @@ import UIKit
 
 extension ReloadableViewLayoutAdapter: UICollectionViewDelegateFlowLayout {
```

Comment:
Thanks! I would just clarify that it refers to subclasses and not the implementation in the extension
`Warning: Subclasses that override this method must call super.`